src/scaffoldmaker/utils/tubenetworkmesh.py

```diff
@@ -196,6 +198,8 @@ def __init__(self, networkSegment, pathParametersList, elementsCountAround, elem
         self._elementsCountAcrossMajor = elementsCountAcrossMajor  # includes 2 * elementsCountTransition
         self._elementsCountAcrossMinor = \
             self._elementsCountAround // 2 - elementsCountAcrossMajor + 4 * elementsCountTransition
+        self._elementsCountCoreBoxMajor = self._elementsCountAcrossMajor - 2 * elementsCountTransition
+        self._elementsCountCoreBoxMinor = self._elementsCountAcrossMinor - 2 * elementsCountTransition
         self._elementsCountTransition = elementsCountTransition
 
         # if self._isCore and self._elementsCountTransition > 1:
```

Comment:
I think lines 205 - 206 can be removed, since we are not re-calculating the number of elementsCountAround as we change the number of transition elements.

Reply:
I've switched to passing and using core box minor/major values internally.